### ISSUE TITLE
NPM version

### ISSUE THREAD

**Issue by edemaine** [Ullallulloo/markdown-it-fancy-lists#1]
I'd like to use this package via NPM.  Would you accept a pull request that makes this library runnable in both web browser (like currently) and in Node?  Or should I stick to a fork?

**Comment by mminer237** [comment]
Yeah, sure thing. I've just never used Node myself. That sounds like a good idea.
